src/widget/range/range-widget.scss

```diff
@@ -389,3 +389,56 @@ $green: rgb(60, 182, 67);
         }
     }
 }
+
+.or-appearance-rating {
+
+    .rating {
+        display: inline-block;
+        width: 100%;
+        height: auto;
+    }
+
+    input {
+        &:not([type=radio]) {
+            opacity: 0; 
+            height: 1px;
+            padding: 0;
+        }
+
+        &[type=radio] {
+            &.star {
+                appearance: none;
+                -moz-appearance: none;
+                -webkit-appearance: none;
+                -ms-appearance: none;
+                display: inline-block;
+                width: 40px;
+                height: 40px;
+                margin-right: 5px;
+                margin-bottom: 0;
+                margin-top: 0;
+                border-width: 10px;
+                border-style: solid;
+                background-color: transparent;
+                background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-6 1 53 50'%3E%0A03 %3Cpath d='m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z' style='fill:%23f76411;'/%3E%0A04%0A%3C/svg%3E");
+                background-size: 20px 20px;
+                background-position: -3px;
+                border-color: #ffffff;
+                border-radius: 0;
+
+                &:checked {
+                    border-color: #ffffff;
+                    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-6 1 53 50'%3E%0A03 %3Cpath d='m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z' style='fill:%23f76411;'/%3E%0A04%0A%3C/svg%3E");
+                }
+            }
+        }
+    }
+
+    /* plain stars, hover behavior */
+    .empty input.star:not(:hover),
+    .star:checked ~ .star:not(:hover) {
+        border-color: #ffffff;
```

Comment:
border-color should be transparent, because repeats have a different background and so do some themes when hovering over the question (and a theme could also provide a custom background for the whole form). Alternatively, and I like this better, is to use padding, and `background-repeat: no-repeat;`, and positioning the background images.

Reply:
code updated